react-accordion-component/src/accordion.jsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+
+class Accordion extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { clicked: '' };
```

Comment:
In state you should store the `id` or 'index` of the topic that was clicked on, the "open" topic.